app/hotels/src/filter/hotelFacilities/HotelFacilitiesFilter.js

```diff
@@ -60,7 +59,7 @@ export default class HotelFacilitiesFilter extends React.Component<
   );
 
   render = () => (
-    <View>
+    <React.Fragment>
```

Comment:
is this needed, can't it just return FilterButton?

Reply:
Yes, it's necessary. There is not only `FilterButton` component.

Reply:
Sorry github UI plays mind tricks on me 🙈